src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/CodeFoldingTests.cs

```diff
@@ -272,6 +272,7 @@ await AssertFoldableBlocksAsync(
             {
                 <h2>s</h2>
             }
+
             """,
```

Comment:
Should the foldable block include that blank line? I would've expected it to stop at the closing curly brace.

Reply:
FUSE handles whitespace differently. See https://github.com/dotnet/razor/issues/10860